compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/draganddrop/DragAndDrop.desktop.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.draganddrop
+
+import androidx.compose.ui.DragData
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.dragData
+import androidx.compose.ui.geometry.Offset
+import java.awt.datatransfer.Transferable
+import androidx.compose.ui.platform.PlatformDragAndDropManager
+import java.awt.dnd.DropTargetDropEvent
+
+/**
+ * Encapsulates the information needed to start a drag-and-drop session from Compose on the desktop.
+ */
+@OptIn(ExperimentalComposeUiApi::class)
+actual class DragAndDropTransferData(
+    val transferable: DragAndDropTransferable,
```

Comment:
Let's mark all properties `@property:ExperimentalComposeUiApi` (we can't mark the class, as it is used in foundation already)

Reply:
That we can do, yes.